### PR TITLE
Update the Model and Serialisers to 3.0.rc02

### PR DESCRIPTION
**Pull request by sebbader**
This PR contains _a lot_ of changes introduced by:

* moving the repositories, therefore also changing the package names, and
* changes in the - not yet released - v3.0.RC02 schemes

Please also note that this is not a final state for 3.0.RC02. Many tests are failing, and I have focused on the JSON part a lot. The other modules compile, but that's it, nothing more.

### CODE REVIEW

**Comment by sebbader** [comment]
@twebermartins and @mjacoby, as discussed today this is the current state of work in direction of 3.0.RC02.

**Comment by twebermartins** [comment]
Merge so that we have the RC02. We know that there are some errors due to missing builds for datafile formats (AML, AASX, XML ..). Issues will be generated to address these errors and will be addressed ASAP.

File: dataformat-json/src/test/resources/assetAdministrationShell.json [hunk 1]
```diff
@@ -1,6 +1,8 @@
 {
   "idShort": "TestAssetAdministrationShell",
-  "description": [
+  "description":
+  {
+    "langStrings":  [
```
Comment (mjacoby) [inline dataformat-json/src/test/resources/assetAdministrationShell.json:5]: This seems not to be standard-compliant (see table 6, page 96 AAS Part 1 v3.0RC02). Description in JSON should be array and not object with additional property "langStrings"

Comment (sebbader) [reply dataformat-json/src/test/resources/assetAdministrationShell.json:5]: Valid point. I have raised an issue at https://github.com/aas-core-works/aas-core-meta/issues/128 as this is the source of the JSON schema that I have imported.

File: dataformat-aasx/src/main/java/org/eclipse/aas4j/v3/rc02/dataformat/aasx/AASXDeserializer.java [hunk 2]
```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.adminshell.aas.v3.dataformat.aasx;
+package org.eclipse.aas4j.v3.rc02.dataformat.aasx;
```
Comment (Smudo99) [inline dataformat-aasx/src/main/java/org/eclipse/aas4j/v3/rc02/dataformat/aasx/AASXDeserializer.java:16]: Would it be possible to go for a package name only containing the target version (v3) and not also including the current working version (in this case rc02)? I don't see a benefit having the sub-package. 

Only the additional effort on the consumer side - when fetching a new version - to update all imports. Also when we come to v3 final, the rc-package will be removed and the build within consumer projects will fail.

Comment (gedack) [reply dataformat-aasx/src/main/java/org/eclipse/aas4j/v3/rc02/dataformat/aasx/AASXDeserializer.java:16]: I would expect that minor versions will **never** contain "breaking" changes.
If this is the case I fully agree with @Smudo99  and would recommend to only have a major version in names.

**Comment by sebbader-sap** [comment]
I am merging this PR into a working branch based on the discussion of the committers meeting from today.

**Pull request merged**